### PR TITLE
RATEST-79:Failing Test:Xxs on Phone number field test

### PR DESCRIPTION
**Pull request by gracebish**
Issue i worked on: https://issues.openmrs.org/browse/RATEST-79

@k-joseph @kdaud All the tests have built successfully.Awaiting review.

### CODE REVIEW

**Comment by kdaud** [comment]
> import org.junit.Ignore;

Is this library still important

File: ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java [hunk 1]
```diff
@@ -31,7 +31,6 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore //ignore due to moving forms functionality
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java:34]: Revert this change  please  thanks

File: ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java [hunk 2]
```diff
@@ -33,7 +33,6 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java:36]: Revert this change to , i think you need to make all your new changes with in this file https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java, otherwise thanks and keep up

File: ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java [hunk 3]
```diff
@@ -31,6 +31,7 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
+    @Ignore //ignore due to moving forms functionality
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/AddFormTest.java]: You do not need to do any thing with this file( class)

**Comment by gracebish** [comment]
I am closing this PR to make a new one without the test.properties file
